.circleci/config.yml

```diff
@@ -50,6 +50,9 @@ jobs:
     resource_class: spaceshard/ax41
```

Comment:
In L47 and L162 (can't comment on them in this PR, so I'm leaving a note on the closest line), there are still mentions of `cargo_cache_key`. Either we can still rely on that value instead of calculating it ourselves in the new script, or we can completely remove it.

Reply:
Yeah, those should be removed completely as they are used only by CircleCI's builtin cache.

Reply:
Removed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/559/commits/6ab3dbaba77c2370e206848f92d146b2d3e75337
Resolving.